README.md

```diff
@@ -1,49 +1,32 @@
-# Setup
+# Shopify Admin, app extensions - Starter code
 
-## CLI
-If you have grabbed this template from the Shopify CLI you should be good to go!
+This repo contains starter code for apps that use _Argo_ to extend the UI of the _Shopify Admin_.       
+For more about _Argo_ and admin extensions, please [read our docs](https://shopify.dev/tutorials/product-subscription-extension-overview).
 
-Start developing your awesome extension.
+## Getting started
```

Comment:
We should encourage people to use the CLI before manually cloning this repo: https://shopify.dev/tutorials/getting-started-product-subscription-extension